example/utils.rs

```diff
@@ -3,16 +3,28 @@ use std::fs;
 use std::io::Result;
 use std::path::Path;
 
-pub const SOCK_ADDR: &str = "unix:///tmp/ttrpc-test";
+#[cfg(unix)]
+pub const SOCK_ADDR: &str = r"unix:///tmp/ttrpc-test";
 
+#[cfg(windows)]
+pub const SOCK_ADDR: &str = r"\\.\pipe\ttrpc-test";
+
+#[cfg(unix)]
 pub fn remove_if_sock_exist(sock_addr: &str) -> Result<()> {
     let path = sock_addr
         .strip_prefix("unix://")
         .expect("socket address is not expected");
 
     if Path::new(path).exists() {
-        fs::remove_file(&path)?;
+        fs::remove_file(path)?;
     }
 
     Ok(())
 }
+
+#[cfg(windows)]
+pub fn remove_if_sock_exist(sock_addr: &str) -> Result<()> {
```

Comment:
warning: unused variable: `sock_addr`
  --> utils.rs:26:29
   |
26 | pub fn remove_if_sock_exist(sock_addr: &str) -> Result<()> {
   |                             ^^^^^^^^^ help: if this is intentional, prefix it with an underscore: `_sock_addr`
   |
   = note: `#[warn(unused_variables)]` on by default